src/components/FinalizeRequests.vue

```diff
@@ -1,47 +1,53 @@
 <template>
-  <div class="user-campaigns" v-for="campaign in campaigns" v-bind:key="campaign.id">
-      <section class="campaign-card">
-        <section>
-        <div class="img-div"><img :src=campaign.image /></div>
-        </section>
-        <div>
-          <div class="campaign-info">
-            <h1> {{campaign.name}}</h1>
-            <p>{{campaign.description}}</p>
-          </div>
-          <div class="requests"><b>Total Request:</b> {{campaign.requests.length}}</div>
-            <div class="requests" v-for="request in campaign.requests" v-bind:key="request.id">
-            <div class="">
-              Description: {{request.description}}
+  <Loading v-if="loading" />
+  <div v-if="!loading">
```

Comment:
This file hasn't had it's async functions moved to the store so I used local state to assign a loading variable.

Reply:
Will eventually have to be moved to the store, especially if we merge components between this one and all campaigns (where create/edit request lives.